src/btree/bt_sync.c

```diff
@@ -597,23 +594,22 @@ __wt_sync_file(WT_SESSION_IMPL *session, WT_CACHE_OP syncop)
             }
 
             /*
-             * If the page was pulled into cache by our read, try to evict it now.
+             * When the timing stress is enabled, perform the leaf page eviction by the checkpoint.
```

Comment:
There is no timing stress available in the 6.0 branch.

Reply:
Thanks - fixed.